guides/manual-installation.md

```diff
@@ -340,364 +304,323 @@ journalctl -fu trento-web
-    -p 4001:4000 \
+The environment variables listed here are examples and should be considered as placeholders. Instead of specifying environment variables directly in the docker run command, it is recommended to use an environment variable file. Store your environment variables in a file and use the --env-file option with the docker run command. Find detailed instructions on how to use an environment variable file with Docker on [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env).
+
+1. Create the secret environment variables:
+
+    ```bash
+    WANDA_SECRET_KEY_BASE=$(openssl rand -out /dev/stdout 48 | base64)
+    TRENTO_SECRET_KEY_BASE=$(openssl rand -out /dev/stdout 48 | base64)
+    ACCESS_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
+    REFRESH_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
+    ```
+
+1.  Install trento-wanda on docker:
+
+    ```bash
+    docker run -d --name wanda \
+        -p 4001:4000 \
+        --network trento-net \
+        --add-host "host.docker.internal:host-gateway" \
+        -e CORS_ORIGIN=localhost \
+        -e SECRET_KEY_BASE=$WANDA_SECRET_KEY_BASE \
+        -e ACCESS_TOKEN_ENC_SECRET=$ACCESS_TOKEN_ENC_SECRET \
+        -e AMQP_URL=amqp://trento_user:trento_user_password@host.docker.internal/vhost \
+        -e DATABASE_URL=ecto://wanda_user:wanda_password@host.docker.internal/wanda \
+        --restart always \
+        --entrypoint /bin/sh \
+        registry.suse.com/trento/trento-wanda:1.2.0 \
+        -c "/app/bin/wanda eval 'Wanda.Release.init()' && /app/bin/wanda start"
+    ```
+
+1. Install trento-web on docker
+
+    > **Note:** Be sure to change the `ADMIN_USERNAME` and `ADMIN_PASSWORD`, these are the credentials that will be required to login to the trento-web UI.
+
+    > **Note:** Add `CHARTS_ENABLED=false` if Prometheus is not installed or you don't want to use the charts feature of Trento.
+
+    ```bash
+    docker run -d \
+    -p 4000:4000 \
+    --name trento-web \
     --network trento-net \
     --add-host "host.docker.internal:host-gateway" \
-    -e CORS_ORIGIN=localhost \
-    -e SECRET_KEY_BASE=$WANDA_SECRET_KEY_BASE \
-    -e ACCESS_TOKEN_ENC_SECRET=$ACCESS_TOKEN_ENC_SECRET \
     -e AMQP_URL=amqp://trento_user:trento_user_password@host.docker.internal/vhost \
-    -e DATABASE_URL=ecto://wanda_user:wanda_password@host.docker.internal/wanda \
+    -e ENABLE_ALERTING=false \
+    -e DATABASE_URL=ecto://trento_user:web_password@host.docker.internal/trento \
+    -e EVENTSTORE_URL=ecto://trento_user:web_password@host.docker.internal/trento_event_store \
+    -e PROMETHEUS_URL='http://host.docker.internal:9090' \
+    -e SECRET_KEY_BASE=$TRENTO_SECRET_KEY_BASE \
+    -e ACCESS_TOKEN_ENC_SECRET=$ACCESS_TOKEN_ENC_SECRET \
+    -e REFRESH_TOKEN_ENC_SECRET=$REFRESH_TOKEN_ENC_SECRET \
+    -e ADMIN_USERNAME='admin' \
+    -e ADMIN_PASSWORD='test1234' \
+    -e ENABLE_API_KEY='true' \
```

Comment:
Same here: When you already recommend to use an environment file, why don't you follow this here? This would make the whole command a lot easier to digest.

Maybe you need to introduce another step.

Reply:
We need to use the same env for SECRET_KEY_BASE and ACCESS_TOKEN_ENC_SECRET for web and wanda.  The REFRESH_TOKEN_ENC_SECRET differs.

We could store them in a configuration file, but I think every experienced docker user would adjust this, as he requires. 

If these steps don't work well with the whole structure of the official docs, we surely can adjust them :+1: